package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "watch": "webpack --watch",
+    "start": "webpack-dev-server --open"
```

Comment:
## Improve functionality
- [ ] great job adding the start script @mimipeshy, but it is not working as t has not been set properly, Would you mind fixing this? Thank you!